keyboards/fjlabs/ad65/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency
+F_CPU = 8000000
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

Reply:
@drashna Can you make this something I can click to accept the suggestion? I had to destroy this fork due to problems with the submodules.

Reply:
No, I can't. Sorry.